analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Verifier.cs

```diff
@@ -190,7 +201,23 @@ private IEnumerable<Compilation> CompileRazor()
                     File.Copy(file, Path.Combine(tempPath, Path.GetFileName(file)));
                 }
 
-                return new[] { workspace.OpenProjectAsync(Path.Combine(tempPath, "EmptyProject.csproj")).Result.GetCompilationAsync().Result };
+                var path = Path.Combine(tempPath, "EmptyProject.csproj");
```

Comment:
Could `EmptyProject` be extracted as a constant?

Reply:
You mean the `EmptyPorject.csproj` string? And what would be the advantage of doing that?

Reply:
I was under the impression the string was used multiple times.
This can be ignored then.